### ISSUE TITLE
[A11y] FAQ List

### ISSUE THREAD

**Issue by rt4914** [oppia/oppia-android#2633]
## Current Output

https://user-images.githubusercontent.com/9396084/107034128-469b9700-67dc-11eb-854c-c589df95510d.mp4

## Issues Identified
The talkback reads this screen as **Oppia** at start, instead it should be **FAQs** . Please add tests for this. (Assigned @Splash1212). 

Note: To understand the above issue(s) completely, it is recommended that you setup Talkback and play with the app keeping it on and that will give you better context.

**Accessibility Guide**: https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide

**Comment by Prince-kushwaha** [comment]
I want to work on this issue

**Comment by rt4914** [comment]
@Prince-kushwaha Assigned you part of it.

**Comment by Sparsh1212** [comment]
@rt4914 There is no activity on this issue from the past 7 days. If this issue is still up for grab, then I would love to fix the first part. Please assign it to me.

**Comment by rt4914** [comment]
Unassigned @Prince-kushwaha because of inactivity for 10 days and assigning it to @Sparsh1212

**Comment by Sparsh1212** [comment]
@rt4914 Shall I follow the same process as we did in the case of [#2656](https://github.com/oppia/oppia-android/pull/2656#issuecomment-776508648)
Also, please tell the value of `label`  for this case.

**Comment by rt4914** [comment]
> @rt4914 Shall I follow the same process as we did in the case of [#2656](https://github.com/oppia/oppia-android/pull/2656#issuecomment-776508648)
> Also, please tell the value of `label` for this case.

@Sparsh1212 Yes but suggest to wait for that PR to get merge first.

**Comment by rt4914** [comment]
@Sparsh1212 You can work on this PR as your original PR has almost been approved.

**Comment by Sparsh1212** [comment]
> @Sparsh1212 You can work on this PR as your original PR has almost been approved.

Please tell the value of `label` for this case.

**Comment by rt4914** [comment]
> > @Sparsh1212 You can work on this PR as your original PR has almost been approved.
> 
> Please tell the value of `label` for this case.

`FAQs`
